### PR TITLE
checkov: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/checkov.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# checkov
+
+> Checkov is a static code analysis tool for infrastructure as code (IaC).
```
Comment (gutjuri) [inline pages/common/checkov.md:3]:
```suggestion
> Checkov is a static code analysis tool for Infrastructure as Code (IaC).
```

File: pages/common/checkov.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+> It is also a software composition analysis (SCA) tool for images and open source packages.
+> More information: <https://www.checkov.io/1.Welcome/Quick%20Start.html>.
+
+- Scan a directory containing IaC (Terraform, Cloudformation, ARM, Ansible, Bicep, Dockerfile, etc):
+
+`checkov --directory {{path/to/IaC_directory}}`
+
+- Scan a IaC file and omit any code block in output:
```
Comment (gutjuri) [inline pages/common/checkov.md:11]:
```suggestion
- Scan a IaC file, omitting code blocks in the output:
```

File: pages/common/checkov.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+`checkov --compact --file {{path/to/IaC_file}}`
```
Comment (gutjuri) [inline pages/common/checkov.md:13]:
```suggestion
`checkov --compact --file {{path/to/file}}`
```

File: pages/common/checkov.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# checkov
+
+> Checkov is a static code analysis tool for infrastructure as code (IaC).
+> It is also a software composition analysis (SCA) tool for images and open source packages.
+> More information: <https://www.checkov.io/1.Welcome/Quick%20Start.html>.
+
+- Scan a directory containing IaC (Terraform, Cloudformation, ARM, Ansible, Bicep, Dockerfile, etc):
+
+`checkov --directory {{path/to/IaC_directory}}`
```
Comment (gutjuri) [inline pages/common/checkov.md:9]:
```suggestion
`checkov --directory {{path/to/directory}}`
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some suggestions.

File: pages/common/checkov.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+> Checkov is a static code analysis tool for Infrastructure as Code (IaC).
+> It is also a software composition analysis (SCA) tool for images and open source packages.
+> More information: <https://www.checkov.io/1.Welcome/Quick%20Start.html>.
+
+- Scan a directory containing IaC (Terraform, Cloudformation, ARM, Ansible, Bicep, Dockerfile, etc):
+
+`checkov --directory {{path/to/directory}}`
+
+- Scan a IaC file, omitting code blocks in the output:
```
Comment (gutjuri) [inline pages/common/checkov.md:11]:
```suggestion
- Scan an IaC file, omitting code blocks in the output:
```

**Review by gutjuri — COMMENTED** [review]
Sorry, missed one :)

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**